src/components/Menu/Menu.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import './Menu.css';
+import MenuLinks from './MenuLinks';
+import { uniqueNoOfDishes, getDish } from './MenuHelper';
+import Dishes from '../Dishes';
+import Cart from '../Cart';
+class Menu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dishes: {},
+      quantities: [],
```

Comment:
the quantities state is related to cart and keeping it out the cartItems state may result in uncontrolled behaviour sometimes, we can include a property named quantity in each items of cart data which would result in controlled behaviour